tests/karate/src/test/java/fe/net/ReplyMethods.java

```diff
@@ -0,0 +1,17 @@
+package fe.net;
+
+import com.intuit.karate.Json;
+
+import java.util.function.Function;
+
+public class ReplyMethods {
+
+  private static final String VALID_REPLY_TEMPLATE = "{\"jsonrpc\":\"2.0\",\"id\":%ID%,\"result\":0}";
+  private static final String VALID_CATCHUP_REPLY_TEMPLATE = "{\"jsonrpc\":\"2.0\",\"id\":%ID%,\"result\":[]}";
+
+  public static Function<String, String> ALWAYS_VALID = msg -> {
+    int id = Json.of(msg).get("id");
+    String template = msg.contains("\"method\":\"catchup\"") ? VALID_CATCHUP_REPLY_TEMPLATE : VALID_REPLY_TEMPLATE;
```

Comment:
again, don't rely on a fixed formatting of the JSON. It'll make the tests very brittle.